### PR TITLE
Fix hash generation for events without parameters/labels

### PR DESCRIPTION
**Pull request by torchello**
FIxes `TypeError: Cannot read property 'forEach' of undefined` error.

### CODE REVIEW

File: packages/models/src/util.js [hunk 1]
```diff
@@ -717,10 +717,14 @@ export function hashEvent(e) {
...
-  );
-  [...e.labels].forEach((l) => content.push(l));
+  if (e.parameters) {
+    e.parameters.forEach((p) =>
+      getStaticPropValues(p).forEach((v) => content.push(v))
+    );
+  }
+  if (e.labels) {
+    [...e.labels].forEach((l) => content.push(l));
+  }
```
Comment (dustinbyrne) [inline packages/models/src/util.js:727]: Was this occurring? `labels` should always exist.

Comment (dustinbyrne) [reply packages/models/src/util.js:727]: The labels need not be copied into an array either, the `forEach` method will exist on the `Set`.

**Comment by kgilpin** [comment]
From what I’ve seen, labels is always Set, never null

**Comment by torchello** [comment]
As discussed with @dustinbyrne removing labels for now.

**Review by dustinbyrne — APPROVED** [review]

**Pull request merged**